runtime/common/src/slots/mod.rs

```diff
@@ -1108,15 +1109,17 @@ mod benchmarking {
 				// Average slot has 4 lease periods.
 				let period_count: LeasePeriodOf<T> = 4u32.into();
 				let period_begin = period_count * i.into();
-				Slots::<T>::force_lease(RawOrigin::Root.into(), para, leaser, amount, period_begin, period_count)?;
+				let origin = T::ForceOrigin::successful_origin();
+				Slots::<T>::force_lease(origin, para, leaser, amount, period_begin, period_count)?;
 			}
 
 			for i in 0 .. max_people {
 				let leaser = account("lease_deposit", i, 0);
 				assert_eq!(T::Currency::reserved_balance(&leaser), T::Currency::minimum_balance());
 			}
 
-		}: _(RawOrigin::Root, para)
+			let origin = T::ForceOrigin::successful_origin();
```

Comment:
Here it is again without try?

Reply:
There is `try_successful_origin` and `successful_origin`, which is automatically implemented as `try_successful_origin().unwrap()`.

In this case, we need the successful origin